docs/source/getting_started.rst

```diff
@@ -1,2 +1,155 @@
+
+* Change settings like the ``ORCHESTRA_PROJECT_API_SECRET`` from ``'CHANGEME'``
+  to more appropriate values.
+
+* Add 3rd party credentials in the "3rd Party Integrations" section so that
+  Orchestra can store files on `Amazon s3 <https://aws.amazon.com/s3/>`_ and use
+  `Google apps <http://apps.google.com>`_ and `Slack <https://slack.com/>`_ to
+  help communicate with expert workers (though these are all optional).
+
+Then, at the bottom of your existing settings file
+(``your_project/your_project/settings.py``), import the orchestra
+settings::
+
+  from .orchestra_settings import setup_orchestra
+  setup_orchestra(__name__)
+
+You'll also need to set up orchestra's URLs, so that Django knows where to
+route users when they view orchestra in the browser. Add the following code
+inside the ``urlpatterns`` variable in
+``your_project/your_project/urls.py``::
+
+    # Registration Views
+    # Eventually these will be auto-registered with the Orchestra URLs, but for
+    # now we need to add them separately.
+    url(r'^orchestra/accounts/',
+        include('registration.backends.default.urls')),
+
+    # Logout then login is not available as a standard django
+    # registration route.
+    url(r'^orchestra/accounts/logout_then_login/$',
+        auth_views.logout_then_login,
+        name='logout_then_login'),
+
+    # Orchestra URLs
+    url(r'^orchestra/',
+        include('orchestra.urls', namespace='orchestra')),
+
+    # Beanstalk Dispatch URLs
+    url(r'^beanstalk_dispatch/',
+        include('beanstalk_dispatch.urls')),
+
+Finally, you'll need to get the database set up. Create your database
+with ``python manage.py migrate``. You'll also want to make sure you have an
+initial worker account set up to try out example workflows. We've provided one
+(username: ``demo``, password: ``demo``) as a fixture which you can load with
+``python manage.py loaddata demo_worker``.
+
+Now Orchestra should be ready to go! If you're confused about any of the above,
+check out our barebones `example project <https://github.com/unlimitedlabs/orchestra/tree/v0.1.0/example_project>`_.
+
+*************
+Run Orchestra
+*************
+
+Now that Orchestra is configured, all that remains is to fire it up! Run your
+django project with ``python manage.py runserver`` (you'll want to switch to
+something more robust in production, of course), and navigate to
+``http://127.0.0.1:8000/orchestra/app`` in your favorite browser.
+
+If you see the Orchestra sign-in page, your setup is working! Logging in as
+the demo user we set up above should show you a dashboard with no available
+tasks.
+
+.. _demo-section:
+
+****************************
+Run the Example Project Demo
+****************************
+
+To give you a feel for what it means to run an Orchestra workflow from end to
+end, we've included a very simple example workflow with two steps, one
+automated and one human. The automated step takes a URL and extracts a random
```

Comment:
Honestly I prefer to describe these steps as automated, but I changed it to machine for consistency with our codebase.